docs/documentation/mission_module/mission_file/segments.rst

```diff
@@ -433,10 +433,30 @@ Typically, it will be used as last segment to compute a reserve based on the Zer
       altitude: 0.
       mach: 0.
 
+Mass loss without fuel consumption
```

Comment:
```suggestion
Mass change without fuel consumption
```

Reply:
Applied in fcc948b001231f531f68d73d034a466598b4bb0f